src/templates/layouts/healthCareLocalFacility/Phone.tsx

```diff
@@ -0,0 +1,154 @@
+import { ParagraphPhoneNumber } from '@/types/drupal/paragraph'
+
+export interface SeparatedPhoneNumber {
+  phoneNumber: string
+  extension: string
+  processed: boolean
+}
+
+export const separatePhoneNumberExtension = (
+  phoneNumber: string
+): SeparatedPhoneNumber | null => {
+  if (!phoneNumber) {
+    return null
+  }
+
+  const phoneRegex =
+    /\(?(\d{3})\)?[- ]*(\d{3})[- ]*(\d{4}),?(?: ?x\.? ?(\d*)| ?ext\.? ?(\d*))?(?!([^<]*>)|(((?!<v?a).)*<\/v?a.*>))/i
```

Comment:
I just pulled this from `content-build`: https://github.com/department-of-veterans-affairs/content-build/blob/cvalarida/deprecate-events-and-eventListings/src/site/filters/liquid.js#L326-L326

Reply:
😵